test/parallel/test-crypto-key-objects.js

```diff
@@ -470,3 +639,45 @@ const privateDsa = fixtures.readKey('dsa_private_encrypted_1025.pem',
     message: "The property 'options.cipher' is invalid. Received undefined"
   });
 }
+
+{
+  // SecretKeyObject export buffer format (default)
+  const buffer = Buffer.from('Hello World');
+  const keyObject = createSecretKey(buffer);
+  assert(buffer.equals(keyObject.export()));
+  assert(buffer.equals(keyObject.export({})));
+  assert(buffer.equals(keyObject.export({ format: 'buffer' })));
+  assert(buffer.equals(keyObject.export({ format: undefined })));
```

Comment:
Nit: When both are buffers, `assert.deepStrictEqual` is better. The main reason is that `assert(buffer.equals(...))` won't show the differences between the buffers when they are not equal.

Reply:
✅